content/root/quick-start-guide.textile

```diff
@@ -489,15 +511,11 @@ blang[default].
 blang[python,php].
   The <span lang="python">Python</span><span lang="php">PHP</span> SDK is only available with a REST interface. You will need to use one of the realtime libraries to create or manage a connection to Ably. Use the language selector above to select a language with realtime support.
 
-bc[javascript](code-editor:quick-start-guide/close). ably.connection.close();
-ably.connection.on('closed', () => {
-  alert('Closed the connection to Ably.');
-});
+bc[javascript]. ably.close(); // resolves synchronously
```

Comment:
I think it's a bit confusing to use the word 'resolves' here, that kind of implies that the method returns a `Promise` but in this case it doesn't. I would suggest changing it to either 'runs synchronously' or 'executes synchronously'

Reply:
OK, changed to "runs".